CONTRIBUTING.md

```diff
@@ -55,4 +58,130 @@ We prefer all communications to be in English.
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
+
+# Develop for CBL-Mariner
+When starting to develop for CBL-Mariner, use the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo. This repository guides developers on using CBL-Mariner's tools to customize or add new packages or images. Once you have confirmed your change builds and functions as expected, consider whether it should be added to the core repo, [CBL-Mariner](https://github.com/microsoft/CBL-Mariner).
+Please see either our [quickstart](toolkit/docs/quick_start/quickstart.md) or [building instructions](toolkit/docs/building/building.md) for technical instructions on building with CBL-Mariner.
+Please adhere to the [Pull Request guidelines](pull-request-guidelines) when contributing. 
+
+## Packages
+CBL-Mariner packages live in either [SPECS](SPECS) or [SPECS-EXTENDED](SPECS-EXTENDED). CBL-Mariner's core packages (`SPECS`) have full support and coverage. Packages in `SPECS-EXTENDED` are only guaranteed to compile successfully. `SPECS-EXTENDED` can be used as a staging area for iterating on packages with the possiblity of the package being graduated to `SPECS`. 
```

Comment:
I think use of the `core packages` can introduce confusion for a casual reader who may not be aware with our terminology, because we also have a meta package `core-packages.spec` that contains the minimal possible packages ....
```suggestion
CBL-Mariner packages live in either [SPECS](SPECS) or [SPECS-EXTENDED](SPECS-EXTENDED). CBL-Mariner packages that are in the `SPECS` directory have full support and coverage with timely CVE maintenance. Packages in `SPECS-EXTENDED` are only guaranteed to compile successfully. `SPECS-EXTENDED` can be used as a staging area for iterating on packages with the possiblity of the package being graduated to `SPECS`. 
```